### DISCUSSION TITLE
trình thám hiểm có sự cố

### DISCUSSION THREAD

**Discussion by phamduyhao** [community/community#31921] [Other Feature Feedback, Questions, & Ideas]
explorer has problem processing data very slow even can't read, suggest development team to reconsider

**Comment by jfullstackdev** [comment]
I just loaded it by now without delay, even with too many tabs open

![image](https://user-images.githubusercontent.com/47092464/188314256-9f2b4be2-3005-4729-8dca-243b7d3e3839.png)

**Reply by phamduyhao** [reply]
connect remote endpoint and complete API initialization causing delay

**Reply by phamduyhao** [reply]
i want to see blockchain explorer it is very slow due to connect remote endpoint and complete API initialization causing delay
